core/standard/requirements/core/REQ_fc-rel-type.adoc

```diff
@@ -2,5 +2,5 @@
 [width="90%",cols="2,6a"]
 |===
 ^|*Requirement {counter:req-id}* |*/req/core/fc-rel-type* 
-^|A |All links SHALL include the `rel` and `type` link parameters.
+^|A |All links in a `200`-response where `rel` is `self`, `alternate`, `next`, or `prev` SHALL include the `type` link parameter.
```

Comment:
```suggestion
^|A |All links in a `200`-response where `rel` is `self`, `alternate`, `next`, or `prev` SHALL include the `type` link parameter.
```

What about `previous`? As far as I understood @pvretano, `prev` and `previous` are both allowed and act the same.

What about `last` and `first`, if used in OAFeat?

Reply:
Yes, both `prev` and `previous` are registered by IANA - and basically synonyms. We only mention `prev`, since `prev` is what the current HTML standard uses. 

Apparently `previous` was from HTML 4. If you look into the current HTML 4 version, it also uses `prev`, so I would consider `previous` as "outdated".

`first` and `last` can of course be used, too. We do that in our implementation. However, that will typically require using an offset for paging and we had decided to leave the paging approach to the implementation and only require `next` links. See #23.

Reply:
Thanks. This leaves me with a couple of questions, but we can also copy this to a new issue:

1. Should there be a recommendation to use prev oder previous? And that otherwise they are equivilent and all requirements (e.g. type is required) for prev also apply to previous?
2. Why is the type field only required for prev and next? I understand this requirement as follows: It doesn't say anything which links are required (just requiring next is fine). This req seems to just say that the type field is required for the mentioned rel types. If that's the case, I guess the type should also be required for all 5 pagination rel types, right? Or why do we need a type on prev and next, but not on the others?

Reply:
Yes, we should discuss this in a separate issue. My thoughts:

I think you are talking about something like a paging extension where clients could expect `prev`, `first` and `last` links. In such an extension, I would not include `previous`, it is just a complication for clients.

Maybe we should not have included anything, even in examples, about `prev` in Core?